src/quote/Quote.ts

```diff
@@ -13,9 +13,50 @@
 import Ajv from 'ajv'
 import QuoteSchema from './QuoteSchema'
 import Identity from '../identity/Identity'
-import { IQuote, IQuoteAgreement, IQuoteAttesterSigned } from '../types/Quote'
+import {
+  IQuote,
+  IQuoteAgreement,
+  IQuoteAttesterSigned,
+  ICostBreakdown,
+} from '../types/Quote'
 import { hashObjectAsStr, verify } from '../crypto/Crypto'
 
+type CompressedCostBreakdown = [
+  ICostBreakdown['gross'],
+  ICostBreakdown['net'],
+  ICostBreakdown['tax']
+]
+
+export type CompressedQuote = [
+  IQuote['attesterAddress'],
+  IQuote['cTypeHash'],
+  CompressedCostBreakdown,
+  IQuote['currency'],
+  IQuote['termsAndConditions'],
+  IQuote['timeframe']
+]
+
+export type CompressedQuoteAttesterSigned = [
```

Comment:
There is a lot of repetition going on. I didn't find a perfect solution, but what do you think about this one:
```
export type CompressedQuoteAttesterSigned = [
  CompressedQuote[0],
  CompressedQuote[1],
  CompressedQuote[2],
  CompressedQuote[3],
  CompressedQuote[4],
  CompressedQuote[5],
  IQuoteAttesterSigned['attesterSignature']
]
```

Reply:
Yeah, I have updated. Let me know how you feel about it. 

Though it doesn't help with the destructuring the array.

Reply:
it's a bit better. too bad, that the spread isn't working.